### PR TITLE
feat: Added Rating and Toast components

### PR DESCRIPTION
**Pull request by arjunbharti**
Components created:

1. Rating
2. Toast

### CODE REVIEW

File: documentation/index.html [hunk 1]
```diff
@@ -776,6 +777,518 @@ <h3 class="content-heading">Getting Started</h3>
...
+        &lt;/a&gt;
+&lt;/div&gt;</code>
+                        </pre>
+                    </div>
+                    <hr>
+                    <!-- rating component -->
+                    <div class="rating-component">
+                        <p class="content-sub_heading">Rating</p>
+                        <p class="sub-heading-description">Ratings provide insight regarding others' opinions and experiences, and can allow the user to submit a rating of their own.</p>
+                        <p class="sub-heading-description"></p>
```
Comment (SRUJANA-PENUGONDA13) [inline documentation/index.html:1261]: Remove this extra p tag

File: documentation/index.html [hunk 2]
```diff
@@ -776,6 +777,518 @@ <h3 class="content-heading">Getting Started</h3>
...
+                            </div>
+                            <div class="rating" id="rating-4">
+                                <i class="far fa-star fa-4x"></i>
+                            </div>
+                            <div class="rating" id="rating-5">
+                                <i class="far fa-star fa-4x"></i>
+                            </div>
+                        </div>
+                        <p class="sub-heading-description">For ratings, use "rating" as class name in the div tag.</p>
+                        <div class="code-snippet">
```
Comment (SRUJANA-PENUGONDA13) [inline documentation/index.html:1280]: Try to maintain equal spacing between heading to sub-heading and rating-display to code snippet. I felt there is little bit more space between rating-display and code snippet you can reduce that.

File: documentation/index.html [hunk 3]
```diff
@@ -776,6 +777,518 @@ <h3 class="content-heading">Getting Started</h3>
...
+        &lt;/div&gt;
+        &lt;div class=&quot;grid-item&quot;&gt;
+                four
+        &lt;/div&gt;
+&lt;/div&gt;                </code>
+                        </pre>
+                    </div>
+                    <hr>
+                    <!-- toast component -->
+                    <p class="content-sub_heading">Toast</p>
```
Comment (SRUJANA-PENUGONDA13) [inline documentation/index.html:1198]: Why did you kept sub-heading and description outside the toast container but for rating you kept it inside. Better to follow one approach for all components.

**Comment by SRUJANA-PENUGONDA13** [comment]
1. It's better to add links along with each component. As the code get increases complexity will also increases.
2. Try to use any plugins like prettier for alignment of HTML docs. Few code snippets missed complete alignment.

I have added few suggestions once look into it.

**Comment by hrishikeshmane** [comment]
You can write utility classes for flex, justify-content, items-center, etc in styles/utils.css as well. Will help in code reusability.

File: documentation/index.html (hunk 2 above)
Comment (arjunbharti) [reply documentation/index.html:1280]: Noted, will update at the end else there will be so many merge conflicts

File: documentation/index.html (hunk 3 above)
Comment (arjunbharti) [reply documentation/index.html:1198]: Noted, will update it too!

**Review by HarshnaNHaswani — COMMENTED** [review]
Looks good to merge

**Pull request merged**